QUARKUS-2991.md

```diff
@@ -0,0 +1,128 @@
+|                             | opentelemetry-spi                     |                   N/A |             3 |
+|                             | opentelemetry-vertx                   |                     5 |             5 |
+|                             | reactive-messaging-amqp               |                     1 |             1 |
+|                             | reactive-messaging-hibernate-orm      |                     1 |             2 |
+|                             | reactive-messaging-hibernate-reactive |                     1 |             2 |
+|                             | reactive-messaging-kafka              |                     3 |             4 |
+|                             | reactive-messaging-mqtt               |                   N/A |             1 |
+|                             | reactive-messaging-rabbitmq           |                     1 |             1 |
+|                             | reactive-messaging-rabbitmq-dyn       |                     1 |             1 |
+|                             | smallrye-config                       |                    25 |            29 |
+|                             | smallrye-context-propagation          |     30<br>(1 skipped) |            28 |
+|                             | smallrye-graphql                      |                     5 |             5 |
+|                             | smallrye-graphql-client               |                     9 |            10 |
+|                             | smallrye-jwt-oidc-webapp              |                     6 |             6 |
+|                             | smallrye-jwt-token-propagation        |                     9 |             9 |
+|                             | smallrye-metrics                      |                    20 |            20 |
+|                             | smallrye-opentracing                  |                    11 |            11 |
+|                             |                                       |                       |               |
+| Integration tests - missing | Open API 3.1                          |                       |               |
+|                             | Fault Tolerance 4.0                   |                       |               |
+|                             | Jakarta EE 10 Core Profile            |                       |               |
+|                             | Reactive Streams Operators 3.0        |                       |               |
+
+### Identified problematic areas
+
+#### Priority
+
+Telemetry 1.0
+- Currently, the TCK is inactive in Quarkus `main` due to an API incompatibility in the latest MP release.
+  An update of the TCK in MP repository is beyond Quarkus' scope.
+- AIs:
+  - Asses the scope of the test coverage in the Quarkus integration tests.
+  - Asses the existing tests in QE TS.
+  - Develop new integration test scenario in QE TS.
+
```

Comment:
Is Reactive Messaging test coverage sufficient or it should be also considered for TD addition?